mixer/cmd/build.go

```diff
@@ -35,6 +35,10 @@ type buildCmdFlags struct {
 	noPublish  bool
 	keepChroot bool
 	template   string
+
+	// TODO: This deserves to be in the config too.
```

Comment:
Why add this to the config when `--packager` will only be used until we switch to dnf entirely?

Reply:
This is less valuable if the switch is imminent. Will remove it then.